numpyro/handlers.py

```diff
@@ -130,6 +350,20 @@ def identity(x):
 
 
 def param(name, init_value):
+    """
+    Annotate the given site as an optimizable parameter for use with
+    :mod:`jax.experimental.optimizers`. For an example of how `param` statements
+    can be used in inference algorithms, refer to :func:`~numpyro.svi.svi`.
+
+    :param str name: name of site.
+    :param np.ndarray init_value: initial value specified by the user. Note that
+        the onus of using this to initialize the optimizer is on the user /
```

Comment:
I actually meant onus here - basically that the responsibility is on the user to ensure that this is used to initialize the optimizer, since there is no magical way of registering them with the param store.

Reply:
Ah yeah, my vocabulary deficiency :D

Reply:
Thanks for explaining!